config/default-amo.js

```diff
@@ -128,6 +128,6 @@ module.exports = {
   experiments: {
     installButtonWarning: false,
     // See https://github.com/mozilla/addons-frontend/issues/8760.
-    installButtonWarning2: true,
+    installButtonWarning2: false,
```

Comment:
Why not removing these config params?

Reply:
The issue we have is that we need to be careful not to reuse experiment ids, because if we do that, and the cookie from the previous experiment hasn't expired yet, then the old cookie is used, instead of generating a new cookie. This isn't particularly scalable, but I'm not sure how else to deal with the issue for now.

Reply:
Do we have a new experiment scheduled soon?

Reply:
We do not, but I still feel uneasy about removing the ids and thereby creating the possibility of a collision.